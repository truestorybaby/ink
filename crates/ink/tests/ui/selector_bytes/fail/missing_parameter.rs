const _: u32 = ink::selector_bytes!();

fn main() {}
