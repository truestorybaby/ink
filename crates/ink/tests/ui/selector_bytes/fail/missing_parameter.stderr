error: expected string or byte string literal as input: unexpected end of input, expected literal
 --> tests/ui/selector_bytes/fail/missing_parameter.rs:1:16
  |
1 | const _: u32 = ink::selector_bytes!();
  |                ^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `ink::selector_bytes` (in Nightly builds, run with -Z macro-backtrace for more info)
